PROVESFlightControllerReference/test/int/1_lora_passthrough_test.py

```
@@ -0,0 +1,58 @@
+pytestmark = pytest.mark.rf_unsafe
+
+lora = "ReferenceDeployment.lora"
+downlinkDelay = "ReferenceDeployment.downlinkDelay"
+PASSTHROUGH_TTY = os.environ.get("LORA_PASSTHROUGH_TTY")
+READ_WINDOW_S = 20.0
+SETTLE_S = 2.0
+
+
+@pytest.mark.skipif(not PASSTHROUGH_TTY, reason="LORA_PASSTHROUGH_TTY not set")
+def test_lora_downlink_reaches_passthrough(
+    fprime_test_api: IntegrationTestAPI, start_gds
```

Ruff (0.15.12)

[warning] Unused function argument: `start_gds`

(ARG001)